Explorer/Assets/DCL/AvatarRendering/AvatarShape/Systems/AvatarShapeVisibilitySystem.cs

```diff
@@ -19,10 +19,12 @@ public partial class AvatarShapeVisibilitySystem : BaseUnityLoopSystem
     {
         private readonly OutlineRendererFeature? outlineFeature;
         private SingleInstanceEntity camera;
+        private Plane[] planes;
 
         public AvatarShapeVisibilitySystem(World world, IRendererFeaturesCache outlineFeature) : base(world)
         {
             this.outlineFeature = outlineFeature.GetRendererFeature<OutlineRendererFeature>();
+            planes = new Plane[6];
```

Comment:
We could create a constant for this `6` value to explain the meaning of using this specific index of the array.